src/app/workflow/dag/dag.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-unused-variable */
```

Comment:
Spec file is auto-generated.  I'm going to double check if it's a new 1.0.3 CLI thing or just my IDE.